framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -215,10 +215,12 @@ protected boolean isPrimarySubtype(AnnotatedTypeMirror subtype, AnnotatedTypeMir
     /**
      * Compare the primary annotations of {@code subtype} and {@code supertype}.
      *
+     * @param subtype the type that might be a subtype
+     * @param supertype the type that might be a supertype
      * @param annosCanBeEmtpy indicates that annotations may be missing from the typemirror
      * @return true if the primary annotation on subtype {@literal <:} primary annotation on
-     *     supertype for the current top or both annotations are null. False is returned if one
-     *     annotation is null and the other is not.
+     *     supertype for the current top. Returns true if both annotations are null. Returns false
```

Comment:
This method operates on AnnotatedTypeMirrors, so instead of, `if both annotations` this should be if `neither annotated type has a primary annotation`.